src/mito2/src/sst/file.rs

```diff
@@ -134,6 +134,8 @@ pub struct FileMeta {
     /// the default value `0` doesn't means the file doesn't contains any rows,
     /// but instead means the number of rows is unknown.
     pub num_row_groups: u64,
+    /// Max sequence in this file
+    pub max_sequence: u64,
```

Comment:
Maybe we should rename it to `sequence`? It is a global sequence for all rows in the file.

TODO: While ingesting SSTs, we can also set this field to assign a sequence to the whole SST. We can do it in the mito engine so the mito engine can correctly bump the sequence number and assign it to the file meta. @MichaelScofield